Phonebook.Frontend/src/app/pages/settings/settings.component.html

```diff
@@ -1,5 +1,5 @@
 <mat-card class="pb-margin-20">
-  <mat-card-header>
+  <div>
```

Comment:
Was that for a reason? Using `mat-card-header` should achieve the same without extra CSS.